src/components/ScrollableContainer/ScrollableContainer.test.js

```diff
@@ -0,0 +1,112 @@
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+          </BodyUI>
+        }
+        footer={<FooterUI className="TESTING__FOOTER">Footer</FooterUI>}
+      />
+    )
+
+    const containerScroll = container.querySelector('.c-ScrollableContainer')
+    const header = container.querySelector('.ScrollableContainer__Header')
+    const body = container.querySelector('.ScrollableContainer__Body')
+    const footer = container.querySelector('.ScrollableContainer__Footer')
+
+    // Elements present
+    expect(containerScroll).toBeInTheDocument()
+    expect(header).toBeInTheDocument()
+    expect(body).toBeInTheDocument()
+    expect(footer).toBeInTheDocument()
+
+    // Passing custom classnames
+    expect(header.classList.contains('TESTING__HEADER')).toBeTruthy()
+    expect(body.classList.contains('TESTING__BODY')).toBeTruthy()
+    expect(footer.classList.contains('TESTING__FOOTER')).toBeTruthy()
+
+    // Correct layout styles
+    expect(window.getComputedStyle(containerScroll).display).toBe('flex')
```

Comment:
I have not checked this (:sweat_smile:), but maybe this matcher could be used to make those assertions a bit more concise? https://github.com/testing-library/jest-dom#tohavestyle

Reply:
ditto, looks nicer